cdap-ui/app/features/tracker/controllers/results-ctrl.js

```diff
@@ -326,6 +334,10 @@ class TrackerResultsController {
       .$promise
       .then((response) => {
         this.truthMeterMap = response;
+        angular.forEach(this.fullResults, (entity) => {
+          entity.meter = response[entity.entityTypeState][entity.name];
```

Comment:
Would there be a possibility that we don't get a truthmeter for a particular entity? Like no entry in `response`?